ui/src/app/pages/workspace/workspace-share.tsx

```diff
@@ -233,7 +312,11 @@ export const WorkspaceShare = fp.flow(withUserProfile())(
       if (this.state.saving) {
         return;
       }
-      this.setState({ saving: true, workspaceShareError: false });
+      this.setState({
+        saving: true,
+        workspaceShareError: false,
+        workflowRolesErrors: [],
```

Comment:
These two values are not changed, already set in the constructor method. Do we actually need to set them here again?

Reply:
good point - removed